engine/runtime-integration-tests/src/test/java/org/enso/interpreter/node/expression/builtin/error/CatchPanicNodeTest.java

```diff
@@ -15,23 +15,21 @@
 import org.enso.interpreter.runtime.library.dispatch.TypeOfNode;
 import org.enso.test.utils.ContextUtils;
 import org.enso.test.utils.TestRootNode;
-import org.graalvm.polyglot.Context;
 import org.hamcrest.Matchers;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Test;
 
 public class CatchPanicNodeTest {
-  private static Context context;
+  @ClassRule public static final ContextUtils ctxRule = ContextUtils.createDefault();
   private static CatchPanicNode catchPanicNode;
   private static HostValueToEnsoNode hostValueToEnsoNode;
   private static TestRootNode testRootNode;
 
   @BeforeClass
   public static void initContextAndData() {
-    context = ContextUtils.createDefaultContext();
-    ContextUtils.executeInContext(
-        context,
+    ctxRule.executeInContext(
```

Comment:
Is `executeInContext` needed? `BeforeClass` should be already _"in the context"_, shouldn't it?